src/main/frontend/components/ReviewForm.js

```diff
@@ -43,19 +44,16 @@ const ReviewForm = props => {
     setErrors({})
   }
 
+  const setStarRatingValue = (starRatingValue) => {
+    formPayload.starRating = starRatingValue

```

Comment:
Be very careful here! We should never update state directly! We will need to use the `setFormPayload` setter to add the `starRating` property.  We can make use of the spread operator `...` to create a new object based on the current state and this new `starRating` and then set the object in state